### PR TITLE
Typo in _fileno, missing stdio include

### PR DESCRIPTION
**Pull request by joaoantoniocardoso**
Fixing a mistake of #196.

### CODE REVIEW

**Comment by JPTIZ** [comment]
> Fixing a mistake of #196.

What a terrible mistake. How did we let that pass?

File: src/lib/evil/evil_unistd.h [hunk 1]
```diff
@@ -30,9 +30,10 @@ EVIL_API int ftruncate(int fd, off_t size);
 #define X_OK    0       /* execute permission, originally '1', just a bypass here*/
 #define F_OK    0       /* Test for existence.  */
 
-#define STDIN_FILENO __fileno(stdin)
-#define STDOUT_FILENO __fileno(stdout)
-#define STDERR_FILENO __fileno(stderr)
+#include <stdio.h>
```
Comment (Coquinho) [inline src/lib/evil/evil_unistd.h:33]: maybe add it in the beginning of the file with other includes?

Comment (JPTIZ) [reply src/lib/evil/evil_unistd.h:33]: Yes, you're right. Didn't see that it wasn't the beginning of the file.

Comment (joaoantoniocardoso) [reply src/lib/evil/evil_unistd.h]: done

**Pull request merged**